src/components/dialogs/parameters/parameters.jsx

```diff
@@ -437,6 +431,32 @@ export const useParametersBackend = (
         }
     }, [optionalServiceStatus, backendFetchParameters, snackError, studyUuid, type]);
 
+    // we need to call backendFetchParameters when ever the study params change
+    useEffect(() => {
```

Comment:
This is almost perfectly the same effect as the previous one.
Avoid code duplication.